### PR TITLE
OpenStack Octavia FlavorID

### PR DESCRIPTION
**Pull request by sn4psh0t**
Support for OpenStack Octavia FlavorID parameter

### CODE REVIEW

**Comment by zetaab** [comment]
@sn4psh0t could you run `./hack/update-expected.sh` and commit changes to own commit

File: upup/pkg/fi/cloudup/openstack/cloud.go [hunk 1]
```diff
@@ -496,6 +497,9 @@ func buildLoadBalancerClient(c *openstackCloud, spec *kops.OpenstackSpec, provid
 			return fmt.Errorf("error building lb client: %w", err)
 		}
 		lbClient = client
+		if spec.Loadbalancer.FlavorID != nil {
```
Comment (zetaab) [inline upup/pkg/fi/cloudup/openstack/cloud.go:500]: hmm is these 3 rows really needed? I do not see where c.flavorID is used?

**Review by zetaab — CHANGES_REQUESTED** [review]

File: upup/pkg/fi/cloudup/openstack/cloud.go (hunk 1 above)
Comment (sn4psh0t) [reply upup/pkg/fi/cloudup/openstack/cloud.go:500]: @zetaab you're right: openstackCloud struct should not be influenced here, sorry for the oversight. Can I push another commit reverting the changes to cloud.go back?

Comment (zetaab) [reply upup/pkg/fi/cloudup/openstack/cloud.go:500]: Just make new commit to remove that

Comment (sn4psh0t) [reply upup/pkg/fi/cloudup/openstack/cloud.go]: done @zetaab

**Review by zetaab — APPROVED** [review]
/approve
/lgtm

Thank you!

**Pull request merged**